### ISSUE TITLE
TypeError: text.split is not a function ...

### ISSUE THREAD

**Issue by LizanLycan** [pavelvlasov/mongoose-search-plugin#7]
Node version: v6.9.4

I'm using Electron cross platform.

ERROR: 

> TypeError: text.split is not a function at AggresiveTokenizer.tokenize ...
> at stemmer.tokenizeAndStem ... 
> at Function.schema.statics.search (...\mongoose-search-plugin\index.js: 36:26) ... 

I probe it in a new Model in the database, and I don't use the `.setKeywords`

**Comment by BobCaSUaL** [comment]
Reviewing the code, the problem seem to be related with the [natural](https://github.com/NaturalNode/natural) library.
The problem persist after putting the
```
 Model.setKeywords(function(err) {
    // ...
  });
```
at the bottom of all models that use this plugin ?

**Comment by LizanLycan** [comment]
Yeah, the problem persist. Everything used with the plugin is directly in the main process. Find a solution using $ regex from Mongo, I have not used the plugin again, sorry.
